src/details.js

```diff
@@ -0,0 +1,64 @@
+import "./styles.css";
```

Comment:
To dużo mniej istotne i mniej uniwersalne niż moje uwagi dotyczące nazwy klasy, ale osobiście doradzałbym nazwanie arkusza stylów powiązanego z klasą/komponentem nazwą, która nawiązywałaby do komponentu, dla którego został rzeczony arkusz stworzony (czyli jeśli komponent opierałby się na klasie WeatherDetailsComponent w pliku weatherDetailsComponent.js, to style nazwałbym sobie weatherDetailsComponent.css, weatherDetailsStyle.css czy weatherDetailsComponentMainStylesheet.css (gdybym chciał mieć naprawdę długą nazwę :wink: ).